### DISCUSSION TITLE
Images not loaded when changing a repository from private to public.

### DISCUSSION THREAD

**Discussion by sh3bu** [community/community#61907] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

## Issue -

The images in the repository when the repository is private looks like `https://private-user-images.githubusercontent.com/67383098/244668004-bc61e96a-9a21-4218-850b-abd73f9f8a08.png`.

But when we convert the same repository to public, as expected the repository's contents are made public but the issue is that GitHub doesn't properly render the image. The exact cause for the error is not known.I think the url of the image is not changed when we change the repository's visibility.

On clicking the image, it shows this error

![image](https://github.com/community/community/assets/67383098/1445d33c-ad2f-4931-8d47-ce080281e27c)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by faisalnazir7** [comment]
Try this :

Log Out and Log In Again

Try a Different Browser or Device

Clear Browser Cache

**Reply by sh3bu** [reply]
I beleive it is not a problem on client side. It has to be fixed at the server side.
